corehq/apps/es/case_search.py

```diff
@@ -278,6 +279,10 @@ def external_id(external_id):
     return filters.term('external_id', external_id)
 
 
+def indexed_on(gt=None, gte=None, lt=None, lte=None, eq=None):
+    return filters.date_range('@indexed_on', gt=None, gte=None, lt=None, lte=None, eq=None)
```

Comment:
I was going to comment that this shouldn't have an `@` but I see this is another example of where we are inconsistent :disappointed: :

- case index: inserted_at
- form index: inserted_at
- app index: @indexed_on
- case search index: @indexed_on

Reply:
Yuck.  Yeah, it's especially annoying that the case index and the case search index use different names, because we'll have to be sure to use the proper filter when switching from one to the other.

FWIW the SolTech team decided to drop the `@`s from the serialization used in this API, so a future PR will do that (not that it affects this line)